server-nodejs/model.js

```diff
@@ -166,7 +166,8 @@ module.exports = function Model()
 		debug('update nodes: ', ids);
 		debug('update keys: ', keys);
 		var keysToUnset = {},
-			keysToSet = {};
+			keysToSet = {},
```

Comment:
if we want to stick to the Crockford convention as mentioned in #99 we now prefer one variable declarative statement for each